example/WORKSPACE

```diff
@@ -41,24 +42,51 @@ load("@rules_haskell//haskell:cabal.bzl", "stack_snapshot")
 # Haskell dependencies and toolchain
 ######################################
 
+load("@io_tweag_gazelle_cabal//:config_settings/setup.bzl", "config_settings")
+
+config_settings(name = "config_settings")
+
+load("@config_settings//:info.bzl", "ghc_version")
 load("@io_tweag_gazelle_cabal//:defs.bzl", "gazelle_cabal_dependencies")
 
 gazelle_cabal_dependencies()
 
 stack_snapshot(
     name = "stackage",
+    components =
+        {
+            "tasty-discover": [
+                "lib",
+                "exe:tasty-discover",
+            ],
+        } if ghc_version == "8.10.7" else {
+            "tasty-discover": [
+                "lib",
+                "exe:tasty-discover",
+            ],
+        },
+    local_snapshot = "@io_tweag_gazelle_cabal//:snapshot-" + ghc_version + ".yaml",
     packages = [
+        "Cabal",  #keep
+        "base",
+        "directory",  #keep
+        "filepath",  #keep
+        "inspection-testing",
         "json",  # keep
-        "path",  # keep
-        "path-io",  #keep
+        "mtl",
+        "tasty",
+        "tasty-discover",
+        "tasty-hunit",
+        "text",
+        "void",
```

Comment:
Is it necessary to add all of these new dependencies and base? I think it is supposed to happen as a result of calling `gazelle update-repos`.